migrations/5_remove_holding_accounts.coffee

```diff
@@ -0,0 +1,83 @@
+Settings = require "settings-sharelatex"
+mongojs = require("mongojs")
+ObjectId = mongojs.ObjectId
+db = mongojs(Settings.mongo.url, ['users', 'projects'])
+async = require "async"
+
+module.exports = HoldingAccountMigration = 
+	DRY_RUN: true
+
+	findHoldingAccounts: (callback = (error, users) ->) ->
+		db.users.find({holdingAccount: true}, {holdingAccount: 1, email: 1}, callback)
+
+	deleteUserProjects: (user_id, callback = (error) ->) ->
+		# Holding accounts can't own projects, so only remove from 
+		# collaberator_refs and readOnly_refs
+		console.log "[Removing user from projects]", user_id
+		db.projects.find {
+			$or: [
+				{collaberator_refs: user_id},
+				{readOnly_refs: user_id}
+			]
+		}, { collaberator_refs: 1, readOnly_refs: 1 }, (error, projects = []) ->
+			return callback(error) if error?
+			jobs = projects.map (project) ->
+				(cb) ->
+					console.log "[Removing user from project]", user_id, JSON.stringify(project)
+					if !project._id?
```

Comment:
if there is a chance there is no id is there not a chance there is also no project?